index.d.ts

```diff
@@ -209,6 +210,11 @@ export interface VideoProperties extends ViewProps {
     onPlaybackStalled?(): void;
     onPlaybackResume?(): void;
     onPlaybackRateChange?(data: OnPlaybackRateData): void;
+    onRemoteTriggeredSkipBack?() : void;
+    onRemoteTriggeredSkipForward?() : void;
+    onRemoteTriggeredPause?() : void;
+    onRemoteTriggeredPlay?() : void;
+    onRemoteTriggeredPlayPauseToggle(): void;
```

Comment:
Lock screen inputs now all callback to react-native code directly to control video playback